### DISCUSSION TITLE
T-pot log files fill up to 800GB

### DISCUSSION THREAD

**Discussion by kanowah** [telekom-security/tpotce#1640] [Q&A]
# ⚠️ Basic support information (commands are expected to run as `root`)

- What OS are you T-Pot running on? Latest
- What is the version of the OS `lsb_release -a` and `uname -a`? Debian
- What T-Pot version are you currently using (only **T-Pot 24.04.x** is currently supported)? T-Pot 24.04.x
- What architecture are you running on (i.e. hardware, cloud, VM, etc.)? VM

I am having an issue where the T-POT has filled up over 97% of the memories and now I can't access the dashboard and also the attack map is not working anymore. So, I would like to know if there ais a way to remove or clean up all logs from SSH as I am able to access the SSH only?

![T-POT Memory](https://github.com/user-attachments/assets/d819a988-6e27-49e3-bfc6-5544fac88f30)

**Comment by t3chn0m4g3** [comment]
Please follow the [ReadMe](https://github.com/telekom-security/tpotce?tab=readme-ov-file#log-persistence).

**Reply by kanowah** [reply]
I'm already aware of this configuration, but the issue arise when there is a sudden peak targeting the Honeypot, for example a huge DDoS attack which fill up the server storage to it max. That mean it can filled up the storage in less than 1 day

**Reply by t3chn0m4g3** [reply]
If ddospot is exhausting your resources, I'd recommend to remove it from the `docker-compose.yml`. Otherwise you can only follow the ReadMe and remove logfiles and indexes to free up space.
